client.go

```diff
@@ -9,20 +9,37 @@ import (
 type Client struct {
 	baseURL       string
 	authorization string
+	userAgent     string
 }
 
 const defaultBaseURL = "https://api.fivetran.com/v1"
 const restAPIv2 = "application/json;version=2"
 
+// WARNING: Update Agent version on each release!
+const defaultUserAgent = "Go-Fivetran/0.2.3"
```

Comment:
We might want to have a separate constant with version only.
But it is no necessary at the moment.

Reply:
I think it's not a problem to update version here=) It's the only place we use version in code